raiden-dapp/CHANGELOG.md

```diff
@@ -515,7 +517,7 @@
 - Add link to privacy policy.
 - Add basic transfer screen.
 
-[Unreleased]: https://github.com/raiden-network/light-client/compare/v0.14.0...HEAD
+[unreleased]: https://github.com/raiden-network/light-client/compare/v0.14.0...HEAD
```

Comment:
Nitpick: Intended change?

Reply:
Vow this is the work of my linter in vscode. I dont remember changing this, I will revert this change back and rebase.

Reply:
ok according `prettier` the small `u` is correct, the plugin is markdown complaint since a long time, so I guess its ok